espressif/vscode-esp-idf-extension
# Add validation for existing setup path

## Description

Added validation for esp-idf path, when user selects "Find ESP-IDF in your system" while configuring the extension.

Jira: [VSC-1174](]https://jira.espressif.com:8443/browse/VSC-1174)

## Type of change
- Enhancement

## Steps to test this pull request

- Configure extension:
![image](https://github.com/espressif/vscode-esp-idf-extension/assets/25748706/83c85e72-7790-4774-afee-97e88c1c7eeb)
- Choose "Select ESP-IDF version:

1. Try to do it with a valid esp-idf folder (it should work)
2. Try to do it with an invalid esp-idf folder (error should appear and install button should be disabled)
![image](https://github.com/espressif/vscode-esp-idf-extension/assets/25748706/792ee64d-bcf3-4716-b19a-1ff3b5c07a83)


## How has this been tested?
As described above

**Test Configuration**:
* ESP-IDF Version: 5.2.2
* OS (Windows,Linux and macOS): Windows 11

## Checklist
